tk_toolchain/cmd_line_tools/tk_build_resource/__init__.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import subprocess
+import argparse
+import re
+
+
+def process_import_line(module, import_text):
+    return (
+        f'from {import_text} import {module}\n'
+        f'for name, cls in {module}.__dict__.items():\n'
+        f'    if isinstance(cls, type): globals()[name] = cls\n'
+    )
+
+
+def verify_compiler(compiler):
+    try:
+        print("Verifying compiler: ", compiler)
+        version_compiler = subprocess.check_output([compiler, "--version"], text=True).strip()
+        return True, version_compiler
+    except Exception as error:
+        return False, error
+
+
+def build_qt(compiler, output, py_built_path, import_text):
+    print(f"The import text to replace will be '{import_text}'")
+    output_path = f"{py_built_path}/{output}.py"
+    with open(output_path, 'w') as output_file:
+        subprocess.run(compiler.split(" "), stdout=output_file, check=True)
+    with open(output_path, 'r') as file:
+        content = file.read()
+    content = re.sub(r'^from PySide2.QtWidgets(\s.*)?$', '', content, flags=re.MULTILINE)
+    content = re.sub(r'^\s*$', '', content, flags=re.MULTILINE)
+    content = re.sub(
+        r'^from PySide2\.(\w+) import (.*)$',
+        lambda m: process_import_line(m.group(1), import_text),
+        content,
+        flags=re.MULTILINE
+    )
+    content = content.replace('from PySide2 import', f'from {import_text} import')
+    with open(output_path, 'w') as file:
+        file.write(content)
+
+
+def build_ui(compiler, qt_ui_path, py_built_path, import_text, name):
```

Comment:
Let's rename this argument to match `build_qt` method signature.

```suggestion
def build_ui(compiler, qt_ui_path, py_built_path, import_text, output):
```